src/Testcontainers/Containers/Modules/Databases/MongoDbTestcontainer.cs

```diff
@@ -23,5 +27,26 @@ public override string ConnectionString
       => string.IsNullOrEmpty(this.Username) && string.IsNullOrEmpty(this.Password)
         ? $"mongodb://{this.Hostname}:{this.Port}"
         : $"mongodb://{this.Username}:{this.Password}@{this.Hostname}:{this.Port}";
+
+    public async Task<ExecResult> ExecMongoScriptAsync(string script, CancellationToken ct = default)
+    {
+      var tempScriptFile = this.GetTempScriptFile();
+
+      await this.CopyFileAsync(tempScriptFile, Encoding.Default.GetBytes(script), 493, 0, 0, ct)
+                      .ConfigureAwait(false);
+
+      return await this.ExecAsync(new[] { "mongo", "db", tempScriptFile }, ct);
```

Comment:
Can't we just use the new MongoDB Shell (`mongosh`) and remove `ExecMongoScriptAsync`?

Reply:
Absolutely. This was just in case there was a desire to support the legacy shell while it is still available. Will remove, thanks.